### PR TITLE
APS spider: use LastRunStore spider

### PR DESCRIPTION
**Pull request by vbalbp**
Signed-off-by: Victor Balbuena <vbalbp@gmail.com>

### CODE REVIEW

File: hepcrawl/spiders/aps_spider.py [hunk 1]
```diff
@@ -199,3 +213,6 @@ def _get_authors_and_collab(self, article):
 
     def _file_name_from_url(self, url):
         return "{}.xml".format(url[url.rfind('/') + 1:])
+
+    def make_file_fingerprint(self, set_):
+        return u'metadataPrefix={}&set={}'.format(self.format, set_)
```
Comment (michamos) [inline hepcrawl/spiders/aps_spider.py]: where is the `format` coming from? do you really need that part? it seems OAI-PMH specific. Are you sure this really works?

File: hepcrawl/spiders/aps_spider.py [hunk 2]
```diff
@@ -41,43 +42,48 @@ class APSSpider(StatefulSpider):
     Example:
         Using the APS spider::
 
-            $ scrapy crawl APS -a 'from_date=2016-05-01' -a 'until_date=2016-05-15' -a 'set=openaccess'
+            $ scrapy crawl APS -a 'from_date=2016-05-01' -a 'until_date=2016-05-15' -a 'sets=openaccess'
 
     .. _See documentation here:
         http://harvest.aps.org/docs/harvest-api#endpoints
+
+    Journals are not supported as a parameter anymore for APS Spider
```
Comment (michamos) [inline hepcrawl/spiders/aps_spider.py]:
```
Note:
    Selecting specific journals is not supported for technical reasons as it's incompatible with the way the last run time is stored.
```

File: hepcrawl/spiders/aps_spider.py [hunk 3]
```diff
@@ -41,43 +42,47 @@ class APSSpider(StatefulSpider):
...
+    @property
+    def url(self):
+        params = {
+            'sets': self.set,
+            'from_date': self.from_date or self.resume_from(set_=self._set),
+            'until_date': self.until_date,
+            'per_page': self.per_page,
+            'date': self.date
+        }
+        return furl(APSSpider.aps_base_url).add(params).url
```
Comment (michamos) [inline hepcrawl/spiders/aps_spider.py]: are the `None` checks that were done previously useless?

Comment (vbalbp) [reply hepcrawl/spiders/aps_spider.py]: I tried it out and it does matter, so changing it back to the old way with the checks

**Pull request merged**